src/components/OrganizationSaas/Dashboard/Layout.tsx

```diff
@@ -11,25 +11,83 @@ import {
   Flex,
   Heading,
   IconButton,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Select,
+  Text,
   useDisclosure,
   VStack,
 } from '@chakra-ui/react'
 import { OrganizationAvatar, OrganizationName } from '@vocdoni/chakra-components'
 import { OrganizationProvider, useClient } from '@vocdoni/react-providers'
-import { useTranslation } from 'react-i18next'
+import { Trans, useTranslation } from 'react-i18next'
 import { Outlet, useLocation } from 'react-router-dom'
 import { HSeparator } from '~components/Auth/SignIn'
 import DarkModeToggle from '~src/themes/saas/components/DarkMode'
-import Wrapper from '~src/themes/saas/components/wrapper'
+import PricingCard from '~src/themes/saas/components/Saas/PricingCard'
+import Wrapper from '~src/themes/saas/components/Saas/Wapper'
 import useDarkMode from '~src/themes/saas/hooks/useDarkMode'
 import { Logo } from '~theme/icons'
 import OrganizationDashboardMenu from './Menu'
 import Settings from './Settings'
 
+const Cards = [
```

Comment:
To add translation to this array  you will need to use the `useTranslation` hook, so the array should be declared inside a component.

Also, it should to be in lowecase (`const cards`). Another good practice is to define the inner object type (to ensure that all objects have the same attributes), something like:

```ts
type CardProps = {
popular: boolean
title: string
// etc
}
const cards: CardProps[] = [
//...
]
```

Reply:
Okay, Aulet told me that he would review the translations later. For now, I've left it so that it looks good visually

Reply:
But you can just do like this

```
t('key.for.translation', { defaultValue: 'String you want to show' })
```

Then its easy to create the translations

Reply:
18fe74e